src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -120,6 +120,15 @@
       <Context>ProjectSubscriptionService;BrowseObject</Context>
     </PropertyPageSchema>
 
+    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)AnalyzerReference.xaml">
+      <Context>Project;ProjectSubscriptionService;BrowseObject</Context>
```

Comment:
Hm it might be that Assemblies are also registered by default common targets and they might have other Contexts there... i would not expect CPS hardcode them anyhow..

---
In reply to: [76300441](https://github.com/dotnet/roslyn-project-system/pull/373#discussion_r76300441) [](ancestors = 76300441)